Library/Formula/librest.rb

```diff
@@ -0,0 +1,40 @@
+class Librest < Formula
+  desc "librest is a library access web services that claim to be RESTful"
```

Comment:
> a library **to** access

Please also remove “librest is a”: `desc "Library to access …"`. We could also shorten `web services that claim to be RESTful` to `RESTful Web services`.